app/grants/templates/grants/cart/eth.html

```diff
@@ -3,408 +3,285 @@
-                  :class="{inactive: ![0,5,10,15].includes(gitcoinFactorRaw), 'badge-inactive': gitcoinFactorRaw != 5}"
-                  @click="gitcoinFactorRaw=5">
-                  5%</div>
-                <div class="btn btn-sm btn-primary font-title font-weight-normal ml-xl-2 px-4"
-                  :class="{inactive: ![0,5,10,15].includes(gitcoinFactorRaw), 'badge-inactive': gitcoinFactorRaw != 10}"
-                  @click="gitcoinFactorRaw=10">
-                  10%</div>
-                <div class="btn btn-sm btn-primary font-title font-weight-normal ml-xl-2 px-4"
-                  :class="{inactive: ![0,5,10,15].includes(gitcoinFactorRaw), 'badge-inactive': gitcoinFactorRaw != 15}"
-                  @click="gitcoinFactorRaw=15">
-                  15%</div>
-                <div class="btn btn-sm mx-3 btn-primary font-title font-weight-normal ml-xl-2 px-4"
-                  :class="{inactive: ![0,5,10,15].includes(gitcoinFactorRaw), 'badge-inactive': gitcoinFactorRaw != 0}"
-                  @click="gitcoinFactorRaw=0">
-                  0%</div>
-                <div class="d-inline-block">
-                  <span class="font-caption md:pl-3">Custom %</span>
-                  <input style="width: 75px;" class="border btn font-title font-weight-normal ml-xl-2 pl-0 text-center mr-2"
-                    type="number" min="0" max="99" step="any" id="gitcoin-grant-input-amount"
-                    name="gitcoin-grant-input-amount" class="form__input text-center"
-                    :class="{inactive: [0,5,10,15].includes(gitcoinFactorRaw)}" value="5"
-                    v-model="gitcoinFactorRaw"
-                  >
-                  {% include '../heart.html' %}
-                </div>
-              </div>
-              <br/>
 
-              <div class="flex-container text-left grant-header-row black m-1 p-1">
-                <div class="w-100">
-                  <div class="flex-container" style="justify-content: start">
-                    <input type="checkbox" id="include_for_clr" name="include_for_clr" v-model="include_for_clr" style="margin-right:1rem">
-                    <label class="font-smaller-1 darker-gray mb-0 pb-0" for="include_for_clr">
-                      Include this contribution to CLR matching
-                    </label>
-                  </div>
-                </div>
-              </div>
+      <!-- ALL DEVICES: Add comments -->
+      <div class="col-12 mt-3" >
+        <div class="row align-items-center text-muted inputs-height">
+          <div class="flex-shrink-1">
+            <div class="cursor-pointer px-0 col-auto">
+              <input v-model="grant.grant_comments_bool" :id="`comment-${grant.grant_id}-blank`" type="checkbox" class="d-none" :name="`comment-${grant.grant_id}`"
+                :checked="grant.grant_comments_bool || !!grant.grant_comments"
+                :disabled="grant.grant_comments ? '' : disabled"
```

Comment:
Mobile view is much improved over my attempt, thanks!

One note is that after adding a few grants to my cart, then viewing the cart page, I get the the below error in the console which I believe stems from this line

```
Property or method "disabled" is not defined on the instance but referenced during render. Make sure that this property is reactive, either in the data option, or for class-based components, by initializing the property. See: https://vuejs.org/v2/guide/reactivity.html#Declaring-Reactive-Properties.

found in

---> <GrantsCart>
       <Root>
```

Reply:
Thanks Matt!! I've got that fixed 👍